modyn/supervisor/internal/triggers/utils.py

```diff
@@ -4,48 +4,13 @@
 
 import pandas as pd
 import torch
-from evidently.metrics import EmbeddingsDriftMetric
-from evidently.metrics.data_drift import embedding_drift_methods
 from modyn.supervisor.internal.triggers.embedding_encoder_utils import EmbeddingEncoder
 from modyn.supervisor.internal.triggers.trigger_datasets import DataLoaderInfo, FixedKeysDataset, OnlineTriggerDataset
 from torch.utils.data import DataLoader
 
 logger = logging.getLogger(__name__)
 
 
-def get_evidently_metrics(
```

Comment:
moved into the `datadrifttrigger.py` as most functionality is covered automatically by pydantic.